src/coreComponents/mesh/generators/VTKFaceBlockUtilities.cpp

```diff
@@ -53,17 +53,17 @@ class ElementToFace
       CellBlock const & cb = cellBlocks.getGroup< CellBlock >( c );
       auto const & l2g = cb.localToGlobalMapConstView();
 
-      std::map< globalIndex, localIndex > g2l;
+      stdMap< globalIndex, localIndex > g2l;
 
       for( auto l = 0; l < l2g.size(); ++l )
       {
         globalIndex const & g = l2g[l];
-        m_elementToCellBlock[g] = c;
-        g2l[g] = l;
+        m_elementToCellBlock.insert( {g, c} );
+        g2l.insert( {g, l} );
       }
 
-      m_cbe[c] = g2l;
-      m_cbf[c] = cb.getElemToFacesConstView();
+      m_cbe.insert( {c, g2l} );
```

Comment:
I'm  a little confused. I would expect `operator[]` to replace the value and `insert` to not if the key already exists. Probably doesn't matter in this case?